cypress/e2e/tests/migration/applicationinventory/analysis/source_analysis_without_creds.test.ts

```diff
@@ -84,6 +88,34 @@ describe(["@tier1"], "Source Analysis without credentials", () => {
         );
     });
 
+    // Automates Bug https://issues.redhat.com/browse/MTA-3440
+    it("Source analysis on bookserver app with EAP8 target", function () {
+        const Application = new Analysis(
+            getRandomApplicationData("eap8-bookserverApp", {
+                sourceData: this.appData["bookserver-app"],
+            }),
+            getRandomAnalysisData(this.analysisData["EAP8_bookserverapp"])
+        );
+        Analysis.open();
+        Application.create();
+        applicationsList.push(Application);
+        cy.wait("@getApplication");
+        cy.wait(2000);
+        Application.analyze();
+        checkSuccessAlert(infoAlertMessage, `Submitted for analysis`);
```

Comment:
We moved the notifications checks to the crud tests so an analysis test doesn't fail if the notification text is changed. So I think this line can be safely removed.

Reply:
Done